5-css/styles.css

```diff
@@ -0,0 +1,165 @@
+body {
+  width: 100%;
+  background-color: #cccccc
+}
+
+nav {
+  width: 100%;
+  line-height: 50px;
+  display: inline;
+  display:inline-block;
+  background-color: white
+}
+
+nav img{
+  padding: 10px;
+  float: right;
+  display:inline-block;
+}
+
+nav ul {
+  list-style-type: none;
+  display:inline-block;
+  vertical-align: middle;
+  padding: 10px;
+}
+
+nav ul li {
+  display: inline;
+  list-style-type: none;
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+}
+
+nav li:not(:first-child):before {
+  content: " | ";
+}
+
+main {
+  float: left;
+  width: 70%;
+  background-color: #cccccc;
+}
+
+main form > div:first-child{
+  background-color: white;
+  border: 2px solid #808080;
+  padding: 10px;
+  margin: 10px;
+}
+
+aside {
+  float: right;
+  width: 30%;
+  background-color: #cccccc;
+}
+
+aside ul {
+  list-style-type: none;
+  background-color: white;
+  text-align: left;
+  border: 2px solid #808080;
+  padding: 10px;
+  margin: 10px;
+}
+
+aside img {
+    height: 100%;
+    width: 100%;
+}
+
+section {
+  display: inline-block;
+  text-align: center;
+}
+
+.options {
+  float: left;
+  width: 50%;
+}
+
+.extras {
```

Comment:
TIP for the future 😉 zachęcam to bardziej opisowych nazw klas, aby było wiadomo czego ona dotyczy